tests/testthat/test-derive_param_framingham.R

```diff
@@ -0,0 +1,52 @@
+input <- tibble::tribble(
+    ~USUBJID,      ~PARAMCD, ~PARAM,                               ~AVAL, ~AVALU,       ~VISIT,    ~AGE, ~SEX, ~SMOKEFL, ~DIABETFL, ~TRTHYPFL,
+    "01-701-1015", "SYSBP",  "Systolic Blood Pressure (mmHg)",     121,   "mmHg",      "BASELINE", 44,   "F",  "N",      "N",       "N",
+    "01-701-1015", "SYSBP",  "Systolic Blood Pressure (mmHg)",     115,   "mmHg",      "WEEK 2",   44,   "F",  "N",      "N",       "Y",
+    "01-701-1015", "CHOL",   "Total Cholesterol (mg/dL)",          216.16,"mg/dL",     "BASELINE", 44,   "F",  "N",      "N",       "N",
+    "01-701-1015", "CHOL",   "Total Cholesterol (mg/dL)",          210.78,"mg/dL",     "WEEK 2",   44,   "F",  "N",      "N",       "Y",
+    "01-701-1015", "CHOLHDL","Cholesteral/HDL-Cholesterol (mg/dL)",54.91, "mg/dL",     "BASELINE", 44,   "F",  "N",      "N",       "N",
+    "01-701-1015", "CHOLHDL","Cholesteral/HDL-Cholesterol (mg/dL)",26.72, "mg/dL",     "WEEK 2",   44,   "F",  "N",      "N",       "Y",
+    "01-701-1028", "SYSBP",  "Systolic Blood Pressure (mmHg)",     119,   "mmHg",      "BASELINE", 55,   "M",  "Y",      "Y",       "Y",
+    "01-701-1028", "SYSBP",  "Systolic Blood Pressure (mmHg)",     101,   "mmHg",      "WEEK 2",   55,   "M",  "Y",      "Y",       "Y",
+    "01-701-1028", "CHOL",   "Total Cholesterol (mg/dL)",          292.01,"mg/dL",     "BASELINE", 55,   "M",  "Y",      "Y",       "Y",
+    "01-701-1028", "CHOL",   "Total Cholesterol (mg/dL)",          246.73,"mg/dL",     "WEEK 2",   55,   "M",  "Y",      "Y",       "Y",
+    "01-701-1028", "CHOLHDL","Cholesteral/HDL-Cholesterol (mg/dL)",65.55, "mg/dL",     "BASELINE", 55,   "M",  "Y",      "Y",       "Y",
+    "01-701-1028", "CHOLHDL","Cholesteral/HDL-Cholesterol (mg/dL)",44.62, "mg/dL",     "WEEK 2",   55,   "M",  "Y",      "Y",       "Y"
+  )
+
+test_that("new observations are derived correctly", {
```

Comment:
We get a fancy validation report now!  Can you start the test with New instead of new.  Just so it looks nicer in the report.  https://github.com/pharmaverse/admiral/releases
![image](https://user-images.githubusercontent.com/10111024/158204516-aa19192e-d866-4560-8dc8-f6419acacbce.png)